tests/runner/platforms/md.cpp

```
@@ -0,0 +1,50 @@
+// text carries no terminator, and nothing is cut to a row: this port is a stream, not a screen.
+void writeToDebugPort(const char * text, std::size_t length, [[maybe_unused]] void * writerData) noexcept {
+  // The port ends a line on a null byte, which is what Debug::PrintNewline() sends. A newline reaching it as-is
+  // would print as one more character, running the whole report into a single line.
+  for (std::size_t index = 0; index < length; ++index)
+    ClownMDSDK::MainCPU::Debug::Print(text[index] == '\n' ? '\0' : text[index]);
+}
+
+} // namespace
+
+// The cartridge linker script enters here: there is no C runtime on this target to call main() for us.
+void _EntryPoint() {
```

Cppcheck (2.21.0)

[style] The function '_EntryPoint' is never used.

(unusedFunction)